server/index.ts

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 // Has to be imported once for build
-import { } from '@strapi/strapi';
+import _, { } from '@strapi/strapi';
```

Comment:
I don't get why you import _ of off `@strapi/strapi`.
Could you elaborate?

Reply:
I just called it _ because it is not being used. However I had to import the default too, to avoid typescript saying strapi is not defined in all of the files.

If you try removing the _ and opening a file such as `lifecycles.ts` you should see an error every time strapi is used saying 'strapi' is not defined. Eslint would also throw some warnings for this.

Reply:
I'm sorry, I just tried this again myself and it does not seem like an issue, I will remove it again